src/app.js

```diff
@@ -0,0 +1,83 @@
+import {promisify} from 'util';
+import {MongoClient} from 'mongodb';
+import {createLogger} from '@natlibfi/melinda-backend-commons';
+
+const setTimeoutPromise = promisify(setTimeout);
+
+export default async function ({mongoUri, mongoDatabaseAndCollections, pollTime}, momentDate) {
+  const logger = createLogger();
+  logger.info('Starting mongo cleaning');
```

Comment:
- Mongo with capital letter
- Should there be more info, or is it clear what mongo cleaning means? :D